python/Ganga/GPIDev/Lib/Job/Job.py

```diff
@@ -184,7 +184,7 @@ class Job(GangaObject):
 
     _schema = Schema(Version(1, 6), {'inputsandbox': FileItem(defvalue=[], typelist=['str', 'Ganga.GPIDev.Lib.File.File.File'], sequence=1, doc="list of File objects shipped to the worker node "),
                                      'outputsandbox': SimpleItem(defvalue=[], typelist=['str'], sequence=1, copyable=_outputfieldCopyable(), doc="list of filenames or patterns shipped from the worker node"),
-                                     'info': ComponentItem('jobinfos', defvalue=None, doc='JobInfo '),
+                                     'info': ComponentItem('jobinfos', defvalue=JobInfo(), doc='JobInfo '),
```

Comment:
The category for the `ComponentItem` in this case is `'jobinfos'` and the default plugin for a category is the first one put into it. Now, since `JobInfo` is the only class in the plugin category `'jobinfos'` it will be the default and therefore the instance created. I'm not a big fan of the plugin system but that is my understanding of it. If this isn't working then there is some deeper problem.